crates/solvers/src/infra/config/dex/file.rs

```diff
@@ -48,6 +49,18 @@ struct Config {
     /// (gas_amount_factor, gas_price_factor, nmb_orders_factor, intercept)
     risk_parameters: (f64, f64, f64, f64),
 
+    /// Back-off growth factor for rate limiting.
+    #[serde(default = "default_back_off_growth_factor")]
+    back_off_growth_factor: f64,
+
+    /// Minimum back-off time in seconds for rate limiting.
+    #[serde(with = "humantime_serde", default = "default_min_back_off")]
```

Comment:
I'm pretty sure we already configure values like this elsewhere in the code and would like to use a consistent parsing. If humantime makes things much easier, we can replace all occurrences in a follow up PR.